303-quarkus-vertx-sql/src/test/resources/application.properties

```diff
@@ -0,0 +1,54 @@
+# App
+app.name=Vertx-sql
+
+# Quarkus
+quarkus.http.port=8082
+quarkus.http.test.port=8081
+
+# Postgresql
+# Database
+%postgresql.app.selected.db=postgresql
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.username=test
+quarkus.datasource.password=test
+quarkus.datasource.reactive.url=postgresql://localhost:5432/amadeus
+quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/amadeus
+
+quarkus.datasource.reactive.max-size=5
+quarkus.datasource.reactive.idle-timeout=1
```

Comment:
As opposed to src/main it's 10, is it intentional?

Reply:
Yes, for test as DBpool connections IDLE it's easier to test (will takes lees time) with 1 sec than 10 sec.